testing/snapcast/APKBUILD

```diff
@@ -38,7 +38,26 @@ build() {
 
 package() {
 	cd "$builddir"
-	make DESTDIR="$pkgdir" installclient installserver
+	make DESTDIR="$pkgdir" installclient installserver || return 1
```

Comment:
You do not need to check for errors. Since recently we set -e in abuild (please remove all return 1 entries below)